Guide.docc/Guidance.md

```diff
@@ -0,0 +1,69 @@
+# Guidance
+
+Get recommendations on how to proceed with migrating your project to the
+Swift 6 language mode.
+
+Enabling complete concurrency checking in a module can yield many data-race
+safety issues reported by the compiler.
+Hundreds, possibly even thousands of warnings are not uncommon.
+When faced with a such a large number of problems,
+especially if you are just beginning to learn about Swift's data isolation
+model, this can feel insurmountable.
+
+**Don't panic.**
+
+Frequently, you'll find yourself making substantial progress with just a few
+changes.
+And as you do, your mental model of how the Swift concurrency system works
+will develop just as rapidly.
+
+## Strategy
+
+Of course, there is no one single approach that will work for all projects.
```

Comment:
I think we tend to avoid "of course" and similar like "it's obvious that..." sine for someone it may not be of course but be a surprise etc, so to not make them feel back avoid that wording?

```suggestion
There is no one single approach that will work for all projects.
```

Reply:
Yes, I actually really dislike this kind of wording too. Not sure how I let that happen.